s3tests/functional/test_s3.py

```diff
@@ -4926,7 +4926,23 @@ def _check_key_content(src, dst):
 
 @attr(resource='object')
 @attr(method='put')
-@attr(operation='check multipart copies with single small part')
+@attr(operation='check multipart copies with single small part - ranges not NotImplemented')
+@attr(assertion='fails')
+@attr('fails_on_aws')
+def test_multipart_copy_small_notimplemented():
+    (src_bucket, src_key) = _create_key_with_random_content('foo')
+    dst_bucket = get_new_bucket()
+    dst_keyname = "mymultipart"
+    size = 1
+    e = assert_raises(boto.exception.S3ResponseError, _multipart_copy, src_bucket.name, src_key.name, dst_bucket, dst_keyname, size)
+    eq(e.status, 501)
+    eq(e.error_code, u'NotImplemented')
```

Comment:
@andrewgaul what worries me is that I've seen some tests stay annotated with `fails_on_rgw` long after they actually started working on RGW.

To deter a repeat of that, I want something that will explicitly fail and need somebody to update the test when the mp copy range code lands in the next release after Jewel. Didn't realize S3Proxy/akashic were using these as-is, because the other s3-tests consumers I'd seen have diverged heavily.

As an alternative, how can we introduce something that says the present RGW will return NotImplement, without affecting non-RGW non-AWS consumers?